### PR TITLE
12.0 overdrawn quantities

### PR DESCRIPTION
**Pull request by lmgr0312**

### CODE REVIEW

File: sale_request/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Sale Request',
+    'summary': 'Create Sale Order From Sale Requests by Product',
+    'version': '12.0.1.0.0',
+    'author': 'Jarsa Sistemas S.A. de C.V., '
+              'Odoo Community Association (OCA)',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/sale-workflow',
+    'license': 'LGPL-3',
```
Comment (nikul-serpentcs) [inline sale_request/__manifest__.py:11]: @lmgr0312 Check license As per README

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Hi @lmgr0312 , thank for this contribution.

- Please follow the migration guidelines in https://github.com/OCA/maintainer-tools/wiki#migration
- you are modifying in this PR 2 different modules `sale_product_set` and `sale_request`
- A better title in the PR will attract more reviewers
- A more detailed description in the PR about what are you doing will attract more reviewers

Please fix it

**Comment by rousseldenis** [comment]
@lmgr0312 What's the status of this ?

**Pull request closed without merging**